src/server/plugins/engine/views/components/fileuploadfield.html

```diff
@@ -0,0 +1,31 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% macro FileUploadField(component) %}
+  {% set model = component.model %}
+  {% set upload = model.upload %}
+
+  <h2 class="govuk-heading-m">Uploaded files</h2>
+  {% if upload.count %}
+    <p class="govuk-body" aria-live="polite">{{upload.successfulCount}} of {{upload.count}} files uploaded</p>
+    <p class="govuk-body">
+      {% if upload.pendingCount %}
+        <a href="." class="govuk-link govuk-link--no-visited-state">Refresh page to update file upload progress</a>
+      {% endif %}
+    </p>
+    {{ govukSummaryList(upload.summary) }}
+
+    <script>
+      // @todo - something more permanent/provide no-js fallback
```

Comment:
This isn't ready for production and will be addressed in https://dev.azure.com/defragovuk/DEFRA-CDP/_workitems/edit/282696